.travis.yml

```diff
@@ -0,0 +1,14 @@
+dotnet: 3.1.402
+env:
+  global:
+  - DOTNET_CLI_TELEMETRY_OPTOUT: 1
+language: csharp
+mono: none
+script:
+- dotnet restore
+- dotnet build
+solution: Andculture.Sitefinity.Testing.sln
+notifications:
+  slack:
```

Comment:
Not 100% sure, but you might need to specify the events that trigger the notification:

```
slack:
    on_failure: always
    on_pull_requests: true
    on_success: always
```